src/components/Checkbox.tsx

```diff
@@ -1,5 +1,12 @@
 import { Checkbox as NBCheckbox, ICheckboxProps, FormControl } from 'native-base'
-import { Control, Controller, ControllerRenderProps, FieldErrors, get } from 'react-hook-form'
+import {
```

Comment:
This would definitely change when #20  would get merged. @MSzalowski I'd suggest we'd first merge this PR and then merge yours.

Reply:
This is not valid anymore, I see that this branch was created from #20 so thats ok!